web-jehlomat/src/screens/Nalezy/Components/Links.tsx

```diff
@@ -25,27 +31,60 @@ const List = styled('ul')({
 
 interface LinksProps {
     syringe: Syringe;
-    onUpdate: () => void;
+    onClose?: () => void;
 }
 
-const Links: FunctionComponent<LinksProps> = ({ syringe, onUpdate }) => (
-    <List>
-        <li>
-            <ActionLink to="/">
-                <span>Zlikvidovat nález</span>
-                <SyringeIcon style={{ width: '20px', height: '20px' }} />
-            </ActionLink>
-        </li>
-        <li>
-            <ActionLink to="/">
-                <span>Upravit</span>
-                <EditIcon style={{ width: '20px', height: '20px' }} />
-            </ActionLink>
-        </li>
-        <li>
-            <Delete syringe={syringe} onDelete={onUpdate} />
-        </li>
-    </List>
-);
+const Links: FunctionComponent<LinksProps> = ({ syringe, onClose }) => {
```

Comment:
`React.FC` is standart 😋

Reply:
Nope, it isn't :D

Reply:
At least in Jehlomat developers used to use FC :)

Reply:
So, why there is such a rule in ESLint/Prettier?

Reply:
what rule?

Reply:
Using short hands over the full names.